webserver/routes/api-service-route.js

```diff
@@ -130,6 +130,35 @@ module.exports.getRoutes = function (storage) {
   });
 
   /**
+   * Get latest service outages.
+   */
+
+  router.get('/services/outages', requireAdmin, function (req, res) {
```

Comment:
This is a reporting route, accesible to all users with permissions to view this service. I think it belongs to ``api-report-route.js``